src/cli/cli_link_metrics.cpp

```diff
@@ -134,8 +144,15 @@ template <> otError LinkMetrics::Process<Cmd("mgmt")>(Arg aArgs[])
     otError                  error;
     otIp6Address             address;
     otLinkMetricsSeriesFlags seriesFlags;
+    bool                     sync  = false;
     bool                     clear = false;
 
+    if (aArgs[0] == "sync")
```

Comment:
@zhanglongxia Do I understand this correctly, the `sync` option can be used with any of the `linkmetrics` commands?

Reply:
No. The command `linkmetrics probe` doesn't have the option `sync`. Based on Jonathan's suggestion, I will change the default mode of all `linkmetrics` commands to `sync`.